src/main/java/uk/co/fivium/els/categories/waterheaters/model/WaterSolarPackagesForm.java

```diff
@@ -0,0 +1,70 @@
+package uk.co.fivium.els.categories.waterheaters.model;
+
+import uk.co.fivium.els.categories.common.StandardTemplateForm50Char;
+import uk.co.fivium.els.model.meta.FieldPrompt;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class WaterSolarPackagesForm extends StandardTemplateForm50Char {
+
+  @FieldPrompt("Declared load profile")
+  @NotBlank(message = "Select a declared load profile")
+  private String declaredLoadProfile;
+
+  @FieldPrompt("Water heating energy efficiency class of the water heater")
+  @NotBlank(message = "Select an energy efficiency indicator")
+  private String heaterEfficiencyRating;
+
+  @FieldPrompt("Water heating energy efficiency class of the package of water heater and solar device")
+  @NotBlank(message = "Select an energy efficiency indicator")
```

Comment:
These error messages for the 2 efficiencies should include the specific type so they can be differentiated more easily.
e.g. 'Select a water heating energy efficiency class'